pkgs/by-name/si/simple-completion-language-server/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  withCitation ? true,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "simple-completion-language-server";
+  version = "0.1.0";
```

Comment:
Maybe use `0-unstable-2025-01-31` because upstream does not update the version and has no git tags.